src/Themes/HackathonTheme/Views/OrchardCore.Users/Account/ChangePassword.cshtml

```diff
@@ -1,33 +1,36 @@
-        <div asp-validation-summary="All" class="text-danger"></div>
-        <div class="form-group">
-            <label asp-for="CurrentPassword" class="asterix control-label">@T["Current password"]</label>
-            <input asp-for="CurrentPassword" class="form-control" aria-describedby="CurrentPassword-described-by" />
-            <span id="CurrentPassword-described-by" asp-validation-for="CurrentPassword" class="text-danger"></span>
-        </div>
-        <div class="form-group">
-            <label asp-for="Password" class="asterix control-label">@T["New password"]</label>
-            <input asp-for="Password" class="form-control" aria-describedby="Password-described-by" />
-            <div id="passwordStrength"></div>
-            <span id="Password-described-by" asp-validation-for="Password" class="text-danger"></span>
-        </div>
-        <div class="form-group">
-            <label asp-for="PasswordConfirmation" class="asterix control-label">@T["New password confirmation"]</label>
-            <input asp-for="PasswordConfirmation" class="form-control"  aria-describedby="PasswordConfirmation-described-by" />
-            <span id="PasswordConfirmation-described-by" asp-validation-for="PasswordConfirmation" class="text-danger"></span>
-        </div>
-        <div class="form-group">
-            <button type="submit" class="btn btn-primary">@T["Change password"]</button>
-        </div>
+<v-card class="px-4">
+  <v-card-title>@T["Change password"]</v-card-title>
+  <v-card-text>
+    <form id="form" asp-controller="Account" asp-action="ChangePassword"
+    asp-route-returnurl="@ViewData["ReturnUrl"]" method="post">
+      @if (ViewData.ModelState.ErrorCount > 0)
+      {
+        <v-alert color="red" type="error">
+          @foreach (var error in ViewData.ModelState.SelectMany(x => x.Value.Errors).Reverse())
+          {
+            <span>@error.ErrorMessage</span>
+          }
+        </v-alert>
+      }
+      <v-row>
+        <v-col cols="12">
+          <v-text-field id="CurrentPassword" type="password" name="CurrentPassword" label="@T["Current password"]" value="@currentPassword?.AttemptedValue" error-messages="@(currentPassword?.Errors.FirstOrDefault()?.ErrorMessage)"></v-text-field>
+          <v-text-field id="Password" type="password" name="Password" label="@T["New password"]" value="@password?.AttemptedValue" error-messages="@(password?.Errors.FirstOrDefault()?.ErrorMessage)"></v-text-field>
+          <v-text-field id="PasswordConfirmation" type="password" name="PasswordConfirmation" label="@T["New password confirmation"]" value="@passwordConfirmation?.AttemptedValue" error-messages="@(passwordConfirmation?.Errors.FirstOrDefault()?.ErrorMessage)"></v-text-field>
+        </v-col>
+      </v-row>
     </form>
-  </section>
-</div>
\ No newline at end of file
+  </v-card-text>
+  <v-card-actions>
+    <v-btn type="submit" form="form" x-large block color="success">@T["Change password"]</v-btn>
```

Comment:
What does the `form="form"` do ? I don't think it's required

Reply:
I put the submit button inside v-card-actions outside of the form's influence. That's the only reason why it's required. I could wrap the entire v-card with form instead?

Reply:
Ah ok, cool. Just wondering.